src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -21,9 +21,20 @@
 import sys
 import threading
 import time
-from typing import DefaultDict, Dict, List, Mapping, Sequence, Set, Tuple
+from typing import (
+    Any,
+    DefaultDict,
+    Dict,
+    Iterable,
+    List,
+    Mapping,
+    Sequence,
+    Set,
+    Tuple,
+)
 
 import grpc
+from grpc._typing import MetadataType
```

Comment:
I wonder why an internal import wasn't caught by pylint. Also AFAIK normally linter nags about importing a class from anything, but `typing`.
Strange.

Reply:
I'm OK with it as is. Though to comply with https://google.github.io/styleguide/pyguide.html#22-imports, it may make sense to do 
```suggestion
from grpc import _typing as grpc_typing
```

And then use grpc_typing.MetadataType